plugins/basic/nexus-repository-maven/src/main/java/org/sonatype/nexus/repository/maven/internal/maven2/Maven2GroupFacet.java

```diff
@@ -10,7 +10,7 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.nexus.repository.maven.internal;
+package org.sonatype.nexus.repository.maven.internal.maven2;
```

Comment:
I think @kellyrob99 is saying that at the moment this module is all about maven2, so the extra package suffix is not necessary here. If in the future we need to make a distinction between future maven formats then it's easy to introduce such a refactoring, or we might put the new format in its own module.